nixos/tests/sx.nix

```diff
@@ -0,0 +1,70 @@
+import ./make-test-python.nix (
+  { pkgs, lib, ... }:
+  {
+    name = "sx";
+    meta.maintainers = with lib.maintainers; [
+      figsoda
```

Comment:
@figsoda I added you as a maintainer for this test since you also are a maintainer of the module, but I can remove you if you don't want.